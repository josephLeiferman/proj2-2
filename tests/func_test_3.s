	add $s0, $zero, 17
	add $s1, $zero, 7
	and $s2, $s0, $s1
	sw $s2, 0x2000