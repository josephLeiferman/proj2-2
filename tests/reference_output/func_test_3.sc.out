0000 0000 0000 0000 0000 0000 0000 0000	0000 0000 0000 0000 0000 0000 0000 0000	0000 0000 0000 0000 0000 0000 0000 0000	0000 0000 0000 0000 0000 0000 0000 0000
0000 0000 0000 0000 0000 0000 0001 0001	0000 0000 0000 0000 0000 0000 0000 0000	0000 0000 0000 0000 0000 0000 0000 0000	0000 0000 0000 0000 0000 0000 0000 0000
0000 0000 0000 0000 0000 0000 0001 0001	0000 0000 0000 0000 0000 0000 0000 0111	0000 0000 0000 0000 0000 0000 0000 0000	0000 0000 0000 0000 0000 0000 0000 0000
0000 0000 0000 0000 0000 0000 0001 0001	0000 0000 0000 0000 0000 0000 0000 0111	0000 0000 0000 0000 0000 0000 0000 0001	0000 0000 0000 0000 0000 0000 0000 0000


